waltid-applications/waltid-digital-credentials/src/style.css

```
@@ -0,0 +1,110 @@
+#status {
+  color: #475569;
+  font-size: 0.95rem;
+}
+
+.log-field {
+  margin: 0;
+  min-height: 240px;
+  max-height: 480px;
+  overflow: auto;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)